### ISSUE TITLE
Warning in logs for unsupported targetSOCreachable

### ISSUE THREAD

**Issue by amenychtas** [tillsteinbach/VWsFriend#613]
This is reported for ID.3 SW version 3.2 while charging at prefered times. 

```
2024-03-09T17:21:21+0000:WARNING:addressable:/vehicles/WVW123/domains/automation/chargingProfiles/nextChargingTimer/targetSOCreachable: An unsupported targetSOCreachable: reachable was provided, known values are [invalid, calculating, unknown] please report this as a bug
2024-03-09T17:21:21+0000:WARNING:addressable:/vehicles/WVW123/domains/chargingProfiles/chargingProfilesStatus/nextChargingTimer/targetSOCreachable: An unsupported targetSOCreachable: reachable was provided, known values are [invalid, calculating, unknown] please report this as a bug
```

**Comment by tillsteinbach** [comment]
Thank you for reporting, will be added in the next release

**Comment by tillsteinbach** [comment]
Should be fixed in 0.24.5
